src/seedu/addressbook/ui/TextUi.java

```diff
@@ -65,7 +46,7 @@ private boolean shouldIgnore(String rawInputLine) {
      * @return true if input line is a comment.
      */
     private boolean isCommentLine(String rawInputLine) {
-        return rawInputLine.trim().matches(COMMENT_LINE_FORMAT_REGEX);
+        return rawInputLine.trim().matches(formatter.getCommentLineFormatRegex());
```

Comment:
This is more pertaining to input formatting.  And you formatter's SR is output formatting only?